doc/release-notes/9860-6.0-release-notes.md

```diff
@@ -0,0 +1,285 @@
+# Dataverse 6.0
+
+This is a platform upgrade release. Payara, Solr, and Java have been upgraded. No features have been added to the Dataverse software itself. Only a handful of bugs were fixed.
+
+Thank you to all of the community members who contributed code, suggestions, bug reports, and other assistance across the project!
+
+## Release Highlights (Major Upgrades, Breaking Changes)
+
+This release contains a major upgrades to core components. Detailed upgrade instructions can be found below.
+
+#### Runtime
+
+- The required Java version has been increased from version 11 to 17.
+    - See PR #9764 for details.
+- Payara application server has been upgraded to version 6.2023.8.
+    - This is a required update.
+    - Please note that Payara Community 5 has reached [end of life](https://www.payara.fish/products/payara-platform-product-lifecycle/)
+    - See PR #9685 and PR #9795 for details.
+- Solr has been upgraded to version 9.3.0.
+    - See PR #9787. for details.
+
+#### Development
```

Comment:
```suggestion
This release contains major upgrades to core components. Detailed upgrade instructions can be found below.

### Runtime

- The required Java version has been increased from version 11 to 17.
    - See PR #9764 for details.
- Payara application server has been upgraded to version 6.2023.8.
    - This is a required update.
    - Please note that Payara Community 5 has reached [end of life](https://www.payara.fish/products/payara-platform-product-lifecycle/)
    - See PR #9685 and PR #9795 for details.
- Solr has been upgraded to version 9.3.0.
    - See PR #9787. for details.

### Development
```

Reply:
Thanks, fixed in eaf97f3add3af82d6ed63016613d1f132dee5e5f